packages/dd-trace/src/config.js

```diff
@@ -13,85 +13,111 @@ class Config {
   constructor (options) {
     options = options || {}
 
-    const enabled = coalesce(options.enabled, platform.env('DD_TRACE_ENABLED'), true)
-    const profilingEnabled = coalesce(options.profiling, platform.env('DD_PROFILING_ENABLED'), false)
-    const debug = coalesce(options.debug, platform.env('DD_TRACE_DEBUG'), false)
-    const logInjection = coalesce(options.logInjection, platform.env('DD_LOGS_INJECTION'), false)
-    const url = coalesce(options.url, platform.env('DD_TRACE_AGENT_URL'), platform.env('DD_TRACE_URL'), null)
-    const site = coalesce(options.site, platform.env('DD_SITE'), 'datadoghq.com')
-    const hostname = coalesce(
+    this.tags = {}
+
+    tagger.add(this.tags, platform.env('DD_TAGS'))
+    tagger.add(this.tags, platform.env('DD_TRACE_TAGS'))
+    tagger.add(this.tags, platform.env('DD_TRACE_GLOBAL_TAGS'))
```

Comment:
@rochdev is this ordering correct? we should be preferring `DD_TAGS` over any other.

Reply:
it sounds like applying DD_TRACE_GLOBAL_TAGS -> options.tags -> DD_TRACE_TAGS -> DD_TAGS would be the correct order?

Reply:
It's correct because of specificity, but it was never implemented correctly, so if we want to drop completely we could change the order. The idea was that you could override global tags with tracing-specific tags.